scripts/int_local.sh

```diff
@@ -16,4 +16,5 @@ export MCLI_PUBLIC_API_KEY="${public_key}"
 export MCLI_PRIVATE_API_KEY="${private_key}"
 export MCLI_ORG_ID="${org_id}"
 
-AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int -coverprofile cover.out
\ No newline at end of file
+AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int -coverprofile cover.out
+AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int/clusterwide... -coverprofile cover.out
```

Comment:
Maybe we don't care about it here, but wouldn't the second run override the coverage output of the previous one?

Reply:
actually can we do merge to one line again but be more specific? I will try

Reply:
merged and tested